src/Copy/BrandCopy.elm

```diff
@@ -0,0 +1,623 @@
+module Copy.BrandCopy exposing (brandCopy, relatedInfo)
```

Comment:
Our convention before was to initialise with emojis for all the string values.
This ensures that no text accidentally gets left in as the wrong brand.

Reply:
i will talk about this with @Nikomus  and get back to the solution

Reply:
I've made a separate issue to do this here: #21